.drone.star

```diff
@@ -1040,6 +1041,7 @@ def buildGlauth():
 		'image': 'webhippie/golang:1.13',
 		'pull': 'always',
 		'commands': [
+      # using ocis-glauth repo because latest glauth doesn't supports the bridge setup
```

Comment:
Please open an issue about this in the ocis repo. What are the symptoms of `doesn't support`?

Reply:
@dpakach @individual-it you guys will know about this ^